js/store/actions.js

```diff
@@ -1,11 +1,42 @@
 import fetchItems from '../api/fetch-items';
+import { geoWatch } from '../api/geolocation';
 
 import {
   EMPTY,
   FETCHED,
   ERRORED,
 } from './constants';
 
+export const watchGeoLocation = () => {
+  return (dispatch) => {
+    dispatch(geoAdquiring);
+    geoWatch((geo) => {
+      dispatch(geoAdquired(geo));
+    },() => {
+      dispatch(geoError);
+    });
+  };
+};
+
+export const geoAdquiring = () => {
+  return {
+    type: 'GEO_ADQUIRING',
+  };
+};
+
+export const geoAdquired = (value) => {
```

Comment:
'Geo' is a bit vague and `acquired` sounds like a one time transaction. Can we be more explicit: `LOCATION_UPDATED`.